src/kv/range.rs

```diff
@@ -1,82 +1,98 @@
 use super::{KeyRange, KeyValue};
-use crate::proto::etcdserverpb;
+use crate::protos::rpc::{
+    RangeRequest, RangeRequest_SortOrder, RangeRequest_SortTarget, RangeResponse,
+};
 use crate::ResponseHeader;
+use protobuf::RepeatedField;
+use utilities::Cast;
 
 /// Request for fetching key-value pairs.
-pub struct RangeRequest {
-    proto: etcdserverpb::RangeRequest,
+pub struct EtcdRangeRequest {
+    /// proto
+    proto: RangeRequest,
 }
 
-impl RangeRequest {
-    /// Creates a new RangeRequest for the specified key range.
+impl EtcdRangeRequest {
+    /// Creates a new `RangeRequest` for the specified key range.
+    #[inline]
+    #[must_use]
     pub fn new(key_range: KeyRange) -> Self {
+        let range_request = RangeRequest {
+            key: key_range.key,
+            limit: 0,
+            revision: 0,
+            sort_order: RangeRequest_SortOrder::NONE,
+            sort_target: RangeRequest_SortTarget::KEY,
+            serializable: false,
+            keys_only: false,
+            count_only: false,
+            min_mod_revision: 0,
+            max_mod_revision: 0,
+            min_create_revision: 0,
+            max_create_revision: 0,
+            ..RangeRequest::default()
```

Comment:
add comma to the end of this line

Reply:
Do you mean add a comma at the end of `..RangeRequest::default()`? It can not, clippy will show `cannot use a comma after the base struct`. Or I misunderstand your meaning?

Reply:
OK